src/wp-includes/theme.php

```diff
@@ -1265,6 +1266,27 @@ function get_header_image_tag( $attr = array() ) {
 		}
 	}
 
+	$attr = array_merge(
+		$attr,
+		wp_get_loading_optimization_attributes( 'img', $attr, 'get_header_image_tag' )
+	);
+
+	// If the default value of `lazy` for the `loading` attribute is overridden
+	// to omit the attribute for this image, ensure it is not included.
```

Comment:
```suggestion
	/*
	 * If the default value of `lazy` for the `loading` attribute is overridden
	 * to omit the attribute for this image, ensure it is not included.
	 */
```

Use multiline comment.

Reply:
Changed in 24e883e